understanding/20/name-role-value.html

```diff
@@ -50,8 +50,13 @@ <h2>Intent of Name, Role, Value</h2>
          focus. The focus state of a control can be programmatically determined, and notifications
          about change of focus are sent to user agents and assistive technology. Other examples
          of user interface control state are whether or not a checkbox or radio button has
-         been selected, or whether or not a collapsible tree or list node is expanded or collapsed.
+         been selected, or whether a collapsible tree view or disclosure widget is in expanded or collapsed state.
```

Comment:
we either need to "an", as in "...is in an expanded..." or else go back to how it was before, which is my preference
```suggestion
         been selected, or whether a collapsible tree view or disclosure widget is expanded or collapsed.
```